repose-aggregator/tests/release-verification/src/docker/deb/Dockerfile

```diff
@@ -40,6 +37,9 @@ RUN cp /release-verification/fake-services/fake-origin/package.json /opt/fake-or
 RUN cp /release-verification/fake-services/fake-origin/app.js /opt/fake-origin/app.js
 RUN sh /release-verification/scripts/fake_origin_install_docker.sh
 
+# This build-arg is used to pass the Repose version number which will be set up in this image.
+ARG REPOSE_VERSION
```

Comment:
I moved this Dockerfile instruction lower in the file to enable reuse of more intermediate images (i.e., layers). Since we don't really reuse intermediate images much, it probably won't make a difference, but there wasn't a reason not to do it.